code/src/Components/Album.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import Buttons from './Buttons.js'
+
+//image needs an URL
+const Album = (props) => {
+  console.log(props)
+    return (
+      <div className="album-card">
+          <div className="image-container">            
+              <img className="album-image" src={props.imgsrc}></img>          
+            <Buttons />
+          </div>        
```

Comment:
When your album title is longer then 1 line it makes your album photo break the grid on top. Your photos should be in line and your text areas should be ready for longer text.

![image](https://user-images.githubusercontent.com/50926159/111636543-e1828a80-87f8-11eb-9491-4b4ed23431c8.png)

Reply:
I believe we've fixed this issue now! :)